### ISSUE TITLE
Say in the README that run-ledger complements a metric tracker rather than replacing one

### ISSUE THREAD

**Issue by kornsour** [kornsour/run-ledger#70]
`Run.log_metric` overwrites:

```python
def log_metric(self, name: str, value: float) -> None:
    """Records a measured metric. Overwrites a prior value for `name`."""
    self._metrics[name] = float(value)
```

So a run stores final values, not curves. That is consistent with what the
ledger is for — identity and comparison, not observability — and it is the
right call. But the README's example puts `log_metric` inside a training loop:

```python
with runledger.Run.start(project="demo", seed=1, params={"lr": 3e-4}) as run:
    for step in range(steps):
        loss = train_step()
        run.log_metric("loss", loss)
```

which reads like step-wise logging and is not. A reader reasonably infers
they can drop their existing tracker, tries it, and finds no curves.

## Why this is an adoption issue rather than a nitpick

Someone evaluating this has to answer "does this replace what I have, or sit
next to it?" before they can decide anything. Right now they have to read the
client source to find out. Getting the answer wrong in either direction is
costly: they either rip out a tracker they still need, or dismiss the tool for
failing at a job it never claimed.

The honest positioning is a strength, not a hedge — "we answer the question
your tracker does not" is a sharper pitch than an implied superset, and it is
what the rest of the project's documentation already does well.

## Direction

- State the relationship in the README: run-ledger records what a run **was**;
  a metric tracker records how it **went**. Most users want both.
- Make the loop example's semantics obvious — either comment that the last
  value per key is what is stored, or move the call out of the loop so the
  example does not imply a series.
- Worth a line on where the boundary sits: what belongs in a fingerprint
  versus what belongs in a dashboard.

**Comment by kornsour** [comment]
Landed on `main` in #72 (191c18d). The README now states the relationship plainly — run-ledger records what a run *was*, a tracker records how it *went* — and the loop example is annotated so `log_metric`'s overwrite semantics are visible at the call site.
